src/db/models/planTag.ts

```diff
@@ -36,6 +36,11 @@ export default defineIDModel({
     optional: {
       comment: { kind: 'checked', type: t.string },
       revisionState: { kind: 'checked', type: PLAN_TAG_REVISION_STATE },
+      type: { kind: 'checked', type: t.string },
+      reportingPeriods: {
```

Comment:
Given that you defined `reportingPeriods` as array of integers in [the migration](https://github.com/UN-OCHA/hpc_service/pull/2744/files#diff-e42de6ac05965509da24c368226d605673b54424d12dc42b219265ddeab55ef8R8), which defaults to an empty string, this column does not belong in `optional`, but in `nonNullWithDefault`.